web/src/components/champ-select/champ-select.vue

```diff
@@ -12,7 +12,7 @@
 <script lang="ts" src="./champ-select.ts"></script>
 
 <style lang="stylus" scoped>
-    @import "style.styl"
+    @import "../../static/style.styl"
```

Comment:
I don't think this is the best place to put the stuff, but I do agree that it either needs to be renamed or moved. Two solutions come to mind:
- Have some `variables.styl` or `settings.styl` or whatever in the root.
- Rename this to `common` or `champ-select` or whatever and keep it in the champ-select folder (since this is only included by champion select components anyway).

I don't think `static` is the correct folder to put this. Right now `static` only contains (third-party) assets, and I'd like to keep it that way (ideally you put something in `static` and never modify it).